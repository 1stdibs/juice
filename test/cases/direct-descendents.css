p em {
  background: #00f;
}
p > em {
  color: #f00;
}
