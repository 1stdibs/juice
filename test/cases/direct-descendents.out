<p><em style="color: red; background: blue;"><em style="background: blue;">woot</em></p>
